velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -696,6 +696,56 @@ TEST_F(DateTimeFunctionsTest, hour) {
   EXPECT_EQ(2, hour("1969-01-01 13:23:00.001"));
 }
 
+TEST_F(DateTimeFunctionsTest, minute) {
+  const auto minute = [&](const StringView timestampStr) {
```

Comment:
const StringView -> std::string_view (remove const and use std::string_view)

Reply:
Updated, thanks.